app/src/ep.config.json

```diff
@@ -2,8 +2,7 @@
   "cortexApi": {
     "path": "/cortex",
     "scope": "vestri",
-    "pathForProxy": "http://localhost:9080",
```

Comment:
@dusanradovanovic I think this file was committed unintentionally. I'm also not seeing your styling change files here.